libraries/chain/include/eosio/chain/block_state.hpp

```diff
@@ -68,6 +68,10 @@ struct finality_data_t {
    uint32_t     final_on_strong_qc_block_num{0};
    digest_type  action_mroot{};
    digest_type  reversible_blocks_mroot{};
+   block_num_type       latest_qc_claim_block_num{0};
+   digest_type          latest_qc_claim_finality_digest;
+   block_timestamp_type latest_qc_claim_timestamp;
+   block_timestamp_type timestamp; // This is the timestamp of the current block.
```

Comment:
I think we are missing the corresponding assignment of values to the struct returned in get_finality_data here : 

https://github.com/AntelopeIO/spring/blob/04f3758e83037eccff7472a8bd7b979e32158591/libraries/chain/block_state.cpp#L384

Reply:
Thanks! Added.